packages/api/cms-api/src/generator/generate-crud-input.ts

```diff
@@ -313,6 +316,32 @@ export async function generateCrudInput(
             console.warn(`${prop.name}: unsupported type ${type}`);
             continue;
         }
+
+        const classValidatorValidators = getMetadataStorage().getTargetValidationMetadatas(metadata.class, prop.name, false, false, undefined);
+        for (const validator of classValidatorValidators) {
+            if (validator.propertyName !== prop.name) continue;
+            const constraints = getMetadataStorage().getTargetValidatorConstraints(validator.constraintCls);
+            for (const constraint of constraints) {
+                const decorator = definedDecorators.find((decorator) => {
+                    return (
+                        // ignore casing since class validator is inconsistent with casing
+                        decorator.getName().toUpperCase() === constraint.name.toUpperCase() ||
+                        // some class validator decorators have a prefix "Is" but not in the constraint name
+                        `Is${decorator.getName()}`.toUpperCase() === constraint.name.toUpperCase()
+                    );
+                });
+                if (decorator) {
+                    const importPath = findValidatorImportPath(decorator.getName(), generatorOptions, metadata);
+                    if (importPath) {
+                        imports.push({ name: decorator.getName(), importPath });
+                        decorators.includes(decorator.getText()) || decorators.unshift(decorator.getText());
+                    }
+                } else {
+                    console.warn(`Decorator import for constraint ${constraint.name} not found`);
+                }
```

Comment:
Shouldn't this throw an error? Or should we just ignore the validator?

Reply:
Good question. For unsupported property types there is just a warning right now and the property gets skipped, that's why I decided to do the same here. But I guess an error would be more obvious and could not be so easily overlooked as if it's a warning 🤔 I think both ways are okay, but I can change that to an error if you think it's better

Reply:
> Good question. For unsupported property types there is just a warning right now and the property gets skipped, that's why I decided to do the same here.

I'd say we keep it like it is to be in line with the rest of the generator.